### PR TITLE
Update message.json

### PR DESCRIPTION
**Pull request by zVapor-Dev**
**Please describe the changes this PR makes and why it should be merged:**
I added the language: Dutch to the message.json file.

**Status and versioning classification:**

### CODE REVIEW

**Comment by xHyroM** [comment]
please change it to Garlic-Team/dev-latest branch

File: src/util/message.json [hunk 1]
```diff
@@ -25,7 +26,8 @@
         "polish": "Musisz posiadać permisję {PERMISSION} aby użyć tej komendy.",
         "indonesian": "Anda harus memiliki izin {PERMISSION} untuk menggunakan command ini.",
         "italian": "Devi avere il permesso {PERMISSION} per usare questo comando.",
-        "french": "Vous devez disposer de l'autorisation {PERMISSION} pour utiliser cette commande."
+        "french": "Vous devez disposer de l'autorisation {PERMISSION} pour utiliser cette commande.",
+        "nederlands": "Je moet de {PERMISSION} permissie hebben om dit command te gebruiken."
```
Comment (S222em) [inline src/util/message.json:30]:
```suggestion
        "nederlands": "Je moet de {PERMISSION} permissie hebben om dit commando te gebruiken."
```

File: src/util/message.json [hunk 2]
```diff
@@ -39,7 +41,8 @@
         "polish": "Klient musi posiadać permisje {PERMISSION} aby użyć tej komendy.",
         "indonesian": "Client harus memiliki izin {PERMISSION} untuk menggunakan command ini.",
         "italian": "Il cliente deve avere il permesso {PERMISSION} per usare questo comando.",
-        "french": "Le client doit disposer de l'autorisation {PERMISSION} pour utiliser cette commande."
+        "french": "Le client doit disposer de l'autorisation {PERMISSION} pour utiliser cette commande.",
+        "nederlands": "De client moet de {PERMISSION} permissie hebben om dit command te gebruiken."
```
Comment (S222em) [inline src/util/message.json:45]:
```suggestion
        "nederlands": "De client moet de {PERMISSION} permissie hebben om dit commando uit te voeren."
```

File: src/util/message.json [hunk 3]
```diff
@@ -53,7 +56,8 @@
         "polish": "Nie posiadasz wymaganych ról aby użyć tej komendy! Potrzebujesz jednej z podanych: {ROLES}",
         "indonesian": "Anda tidak memiliki salah satu peran yang diperlukan untuk menggunakan command ini! Anda memerlukan salah satu peran dari yang berikut: {ROLES}",
         "italian": "Non hai uno dei ruoli richiesti per usare questo comando! Hai bisogno di uno dei seguenti: {ROLES}",
-        "french": "Vous n'avez pas le rôle requis pour utiliser cette commande ! Vous avez besoin d'un des rôles suivants : {ROLES}"
+        "french": "Vous n'avez pas le rôle requis pour utiliser cette commande ! Vous avez besoin d'un des rôles suivants: {ROLES}",
+        "nederlands": "Je hebt geen van de vereiste roles om dit command te kunnen gebruiken! Je hebt een van de volgende rollen nodig: {ROLES}"
```
Comment (S222em) [inline src/util/message.json:60]:
```suggestion
        "nederlands": "Je hebt geen van de vereiste rollen om dit commando te kunnen gebruiken! Je hebt een van de volgende rollen nodig: {ROLES}"
```

File: src/util/message.json [hunk 4]
```diff
@@ -67,7 +71,8 @@
         "polish": "Musisz poczekać {COOLDOWN} sekund przed ponownym użyciem komendy.",
         "indonesian": "Anda harus menunggu {COOLDOWN} lagi sebelum menggunakan command itu lagi.",
         "italian": "Devi aspettare ancora {COOLDOWN} prima di usare questo comando di nuovo.",
-        "french": "Vous devez patienter {COOLDOWN} avant de réutiliser cette commande."
+        "french": "Vous devez patienter {COOLDOWN} avant de réutiliser cette commande.",
+        "nederlands": "Je moet {COOLDOWN} wachten voor je dat command opnieuw kan gebruiken."
```
Comment (S222em) [inline src/util/message.json:75]:
```suggestion
        "nederlands": "Je moet {COOLDOWN} wachten voor je dat commando opnieuw kan gebruiken."
```

File: src/util/message.json [hunk 5]
```diff
@@ -81,7 +86,8 @@
         "polish": "Nie mogę znaleźć komendy {COMMAND}!",
         "indonesian": "Tidak bisa mencari command {COMMAND}!",
         "italian": "Non è stato possibile trovare il comando {COMMAND}!",
-        "french": "Impossible de trouver la commande {COMMAND} !"
+        "french": "Impossible de trouver la commande {COMMAND} !",
+        "nederlands": "Het command {COMMAND} is niet gevonden!"
```
Comment (S222em) [inline src/util/message.json:90]:
```suggestion
        "nederlands": "Het commando {COMMAND} is niet gevonden!"
```

File: src/util/message.json [hunk 6]
```diff
@@ -95,7 +101,8 @@
         "polish": "Możesz uruchomić to polecenie tylko w kanałach tekstowych.",
         "indonesian": "Anda hanya bisa menjalankan command di channel teks saja.",
         "italian": "Puoi usare questo comando solo in canali da testo.",
-        "french": "Vous ne pouvez lancer cette commande que dans les salons textuels."
+        "french": "Vous ne pouvez lancer cette commande que dans les salons textuels.",
+        "nederlands": "Je kan dit command alleen gebruiken in text kanalen."
```
Comment (S222em) [inline src/util/message.json:105]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in text kanalen."
```

File: src/util/message.json [hunk 7]
```diff
@@ -109,7 +116,8 @@
         "polish": "To polecenie można uruchomić tylko w kanałach informacyjnych.",
         "indonesian": "Anda hanya bisa menjalankan command ini di channel berita saja.",
         "italian": "Puoi usare questo comando solo in canali da notizie.",
-        "french": "Vous ne pouvez lancer cette commande que dans les salons d'information."
+        "french": "Vous ne pouvez lancer cette commande que dans les salons d'information.",
+        "nederlands": "Je kan dit command alleen gebruiken in nieuws kanalen."
```
Comment (S222em) [inline src/util/message.json:120]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in nieuws kanalen."
```

File: src/util/message.json [hunk 8]
```diff
@@ -123,7 +131,8 @@
         "polish": "To polecenie można uruchomić tylko w kanałach informacyjnych.",
         "indonesian": "Anda hanya bisa menjalankan command ini di sebuah thread saja.",
         "italian": "Puoi usare questo comando solo in canali da discussione.",
-        "french": "Vous ne pouvez lancer cette commande que dans les fils de discussion."
+        "french": "Vous ne pouvez lancer cette commande que dans les fils de discussion.",
+        "nederlands": "Je kan dit command alleen gebruiken in threads."
```
Comment (S222em) [inline src/util/message.json:135]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in threads."
```

File: src/util/message.json [hunk 9]
```diff
@@ -137,7 +146,8 @@
         "polish": "Tej komendy możesz użyć tylko na kanale NSFW.",
         "indonesian": "Anda hanya bisa menjalankan command ini di sebuah channel NSFW saja.",
         "italian": "Puoi usare questo comando solo in canali da NSFW.",
-        "french": "Vous ne pouvez lancer cette commande que dans les salons NSFW."
+        "french": "Vous ne pouvez lancer cette commande que dans les salons NSFW.",
+        "nederlands": "Je kan dit command alleen gebruiken in nsfw kanalen."
```
Comment (S222em) [inline src/util/message.json:150]:
```suggestion
        "nederlands": "Je kan dit commando alleen gebruiken in nsfw kanalen."
```

File: src/util/message.json [hunk 10]
```diff
@@ -165,7 +176,8 @@
         "polish": "Nie masz już czasu na odpowiedź. Musisz ponownie uruchomić polecenie.",
         "indonesian": "Anda tidak punya waktu lagi untuk menjawab. Anda harus menjalankan command ini lagi.",
         "italian": "Non ti resta nessun tempo per rispondere. Devi usare il comando di nuovo.",
-        "french": "Le temps de réponse est dépassé. Vous devez relancer la commande."
+        "french": "Le temps de réponse est dépassé. Vous devez relancer la commande.",
+        "nederlands": "Je tijd om deze vraag te beantwoorden is op. Voer het command opnieuw uit."
```
Comment (S222em) [inline src/util/message.json:180]:
```suggestion
        "nederlands": "Je tijd om deze vraag te beantwoorden is op. Voer het commando opnieuw uit."
```

File: src/util/message.json [hunk 11]
```diff
@@ -221,6 +236,7 @@
         "polish": "Proszę wybrać jedno z następujących podpoleceń: {choices}.",
         "indonesian": "Silakan pilih salah satu dari sub-perintah berikut: {choices}.",
         "italian": "Seleziona uno dei seguenti sottocomandi: {choices}.",
-        "french": "Veuillez sélectionner l'une des sous-commandes suivantes: {choices}."
+        "french": "Veuillez sélectionner l'une des sous-commandes suivantes: {choices}.",
+        "nederlands": "Kies aub een van de volgende subcommands: {choices}."
```
Comment (S222em) [inline src/util/message.json:240]:
```suggestion
        "nederlands": "Kies aub een van de volgende subcommando's: {choices}."
```

**Review by S222em — CHANGES_REQUESTED** [review]
Could you also change the branch to dev-latest?

**Comment by zVapor-Dev** [comment]
alr I updated the suggestions and changed the branch

**Review by S222em — APPROVED** [review]

**Comment by xHyroM** [comment]
Please resolve conflicts

**Comment by zVapor-Dev** [comment]
ok done

**Comment by S222em** [comment]
Something went wrong here. Make sure the branch you merge with is dev-latest and not the next branch

**Comment by zVapor-Dev** [comment]
> Something went wrong here. Make sure the branch you merge with is dev-latest and not the next branch

On my end it says awaiting approval

**Comment by xHyroM** [comment]
Don't edit package.json

**Comment by zVapor-Dev** [comment]
You asked me to resolve the conflicts so I did. Package.json was one of them

**Review by S222em — CHANGES_REQUESTED** [review]
This is a merge with the package.json from the work in progress next version of commands. We can not merge it in this state. Please change the package.json to its original state from the correct branch.

**Comment by zVapor-Dev** [comment]
But this is the one from gcommands dev

**Comment by S222em** [comment]
> But this is the one from gcommands dev

It's definitely not. This is the correct package.json https://github.com/Garlic-Team/GCommands/blob/latest-dev/package.json

**Comment by zVapor-Dev** [comment]
I’ll have a look when I’m home

**Comment by zVapor-Dev** [comment]
I should've added it now. please re-review

**Review by xhyrom — APPROVED** [review]

**Review by S222em — APPROVED** [review]

**Pull request merged**